app/assets/javascripts/discourse/app/templates/user.hbs

```diff
@@ -452,4 +452,4 @@
       </div>
     </div>
   </DSection>
-</div>
\ No newline at end of file
+</div>
```

Comment:
A rogue newline 🙂

Reply:
I wasn't sure where that came from. :) I thought it may have been helping me out by doing something automatically.